lib/protobuf/protoc/generator.ex

```diff
@@ -51,7 +58,14 @@ defmodule Protobuf.Protoc.Generator do
 
     service_defmodules =
       if "grpc" in ctx.plugins do
-        Enum.map(desc.service, &Generator.Service.generate(ctx, &1))
+        desc.service
+        |> Enum.with_index()
+        |> Enum.map(fn {service, index} ->
+          Generator.Service.generate(
+            Context.append_comment_path(ctx, "6.#{index}"),
```

Comment:
Same question for the hard-coded `6.`. 😄

Reply:
This number is from FileDescriptorProto. Copied from [the upstream code](https://github.com/pseudomuto/protokit/blob/dc6fd1c8003a61bcd77ddc148e4d49946fbf8c23/parser.go#L13).